src/graphql/schema.gql

```diff
@@ -385,6 +391,7 @@ type Space {
   verified: Boolean
   flagged: Boolean
   rank: Float
+  created_at: Int!
```

Comment:
filters like created_at_gte or created_at_lte are more useful to filter, created_at is just a single timestamp.  so it will be hard to get more spaces than 30k limit

Reply:
Filters are in `SpaceWhere` (https://github.com/snapshot-labs/snapshot-hub/pull/726/files#diff-f5d6a46a522635fe64ba4ec435a92609da73330c512801e8c0efd1cd7807a8b0R120), this is just the returned value

Reply:
This confused me 😆 
<img width="202" alt="Untitled 2" src="https://github.com/snapshot-labs/snapshot-hub/assets/15967809/c9049b3a-232e-4011-afeb-6bc9cd67ddbc">